packages/storybook/src/stories/Alert.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import Alert from '@littlespoon/alert/src/Alert'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+export default {
+  title: 'Design System/Alert',
+  component: alert,
+} as ComponentMeta<typeof Alert>
+
+function onClose() {
+  // Empty function to fix lint error
+}
+
+const Template: ComponentStory<typeof Alert> = (args) => <Alert {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  title: 'Alert Title',
+  description: 'Description Copy goes here',
+  variant: 'success',
+  onClose,
+}
+
+export const TitleNotDismissableNoAction = Template.bind({})
```

Comment:
Can we rename this and the one below to `AlertWithClose` and `AlertWithoutClose`?